src/Parts/Presence.php

```diff
@@ -10,7 +10,7 @@ class Presence
     public string $guild_id;
     public string $status;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Activity[]
+     * @var Activity[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)